clusters/app.ci/quayio-pull-through-cache/quayio-pull-through-cache-us-east-1.yaml

```
@@ -0,0 +1,91 @@
+apiVersion: v1
+kind: List
+items:
+- apiVersion: route.openshift.io/v1
+  kind: Route
+  metadata:
+    name: quayio-pull-through-cache-us-east-1
+    namespace: ci
+  spec:
+    to:
+      kind: Service
+      name: quayio-pull-through-cache-us-east-1
+    tls:
+      termination: Edge
+      insecureEdgeTerminationPolicy: Redirect
+- apiVersion: v1
+  kind: Service
+  metadata:
+    name: quayio-pull-through-cache-us-east-1
+    namespace: ci
+  spec:
+    selector:
+      app: quayio-pull-through-cache-us-east-1
+    ports:
+    - port: 5000
+      targetPort: 5000
+    type: ClusterIP
+- apiVersion: apps/v1
+  kind: Deployment
+  metadata:
+    name: quayio-pull-through-cache-us-east-1
+    namespace: ci
+    labels:
+      app: quayio-pull-through-cache-us-east-1
+  spec:
+    replicas: 1
+    selector:
+      matchLabels:
+        app: quayio-pull-through-cache-us-east-1
+    template:
+      metadata:
+        labels:
+          app: quayio-pull-through-cache-us-east-1
+      spec:
```

Suggesting these run on infra nodes so they have less to compete with. 
```suggestion
      spec:
        tolerations:
        - key: node-role.kubernetes.io/infra
          operator: Exists
        nodeSelector: 
          node-role.kubernetes.io: infra
```